### ISSUE TITLE
Switch to Pytorch

### ISSUE THREAD

**Issue by bazylhorsey** [jonra1993/fastapi-alembic-sqlmodel-async#68]
Reasons: nearly all models are supported on pytorch in the Transformer Pipeline feature, much more limited on tensorflow.

**Comment by jonra1993** [comment]
Hello @bazylhorsey it migrates to pytorch in this commit https://github.com/jonra1993/fastapi-alembic-sqlmodel-async/commit/5803ea38ff841e1ae164d282b0f38ed54d2fa314. I decided to use CPU only version to reduce the final container size of computers without gpu.

**Comment by bazylhorsey** [comment]
That's not true, my understanding is tesnorflow-cpu is actually bigger than tensorflow, because tensorflow has only GPU, tensorflow-cpu has GPU+CPU capabilities meaning it's the bigger of the two. Both PyTorch and tensorflow-cpu are about the same size (1GB)
